src/operator/rnn.cc

```diff
@@ -385,7 +382,9 @@ The definition of GRU here is slightly different from paper but compatible with
 })
```

Comment:
I don't think the projection support is clear in the documentation. Could you update the documentation with LSTMP support when projection_size is set? You can refer to https://github.com/apache/incubator-mxnet/blob/62a85f365b819829fedb60116f803e0c0a3c554c/python/mxnet/gluon/contrib/rnn/rnn_cell.py#L197 

Thanks!

Reply:
Sure. Thanks for pointing out that.

Reply:
Done. Please take a review again. Thanks.